pkg/converter/convert_unix.go

```diff
@@ -852,12 +857,6 @@ func LayerConvertFunc(opt PackOption) converter.ConvertFunc {
 			newDesc.Annotations[label.NydusRefLayer] = desc.Digest.String()
 		}
 
-		if opt.Backend != nil {
```

Comment:
Why removing this code block?

Reply:
> Why removing this code block? It's used to push generated blob to backend storage, and it should be irrelevented to this PR?

Yes, I accidentally deleted this code block and it should be saved. Sorry for this.